apis/beacon/states/randao.yaml

```diff
@@ -0,0 +1,63 @@
+get:
+  operationId: "getStateRandao"
+  summary: "Get the RANDAO mix for some epoch."
```

Comment:
don't we want per-slot granularity here?

right now you will call this endpoint with a given `epoch` and its value will change throughout the epoch

if we specify per-slot then infra can cache the value and consumers can be more precise with exactly what value they want

Reply:
You already get that granularity with the `StateId` path param, right? 

So maybe the description here should be more like "Get the RANDAO mix for some epoch in a specified state."

Reply:
right, yeah that makes sense

Reply:
agreed!

Reply:
done!